apps/frontend/src/constants/gasLimits.ts

```diff
@@ -16,7 +16,7 @@ export const GAS_LIMIT = {
   STAKING_INCREASE_STAKE: 450_000,
   STAKING_EXTEND: 450_000,
   STAKING_WITHDRAW: 1_400_000,
-  VESTING_DELEGATE: 450_000,
+  VESTING_DELEGATE: 1_690_000,
```

Comment:
I got two failing txs when I tried the default settings, raise the limit to 2000 000 at least. This is my last successful tx and as you can see, it used almost 2000 000 gas.